README.md

```diff
@@ -223,6 +223,40 @@ dependencies:
           - pytest
 ```
 
+## Naming conventions
+
+With non-trivial collections of files and dependencies, names can look very related and become hard to distinguish. Some general guidelines that will help keep things more understandable:
+
+### Patterns for [RAPIDS build backend](https://github.com/rapidsai/rapids-build-backend)
+
+RBB builds generally include at least two sections:
+
+```
+files:
+  # This dependency brings in RBB
+  py_build_<something>:
+    output: pyproject
+    extras:
+      table: build-system  # This is the thing to match with the py_build_* name
```

Comment:
```suggestion
      table: build-system
```

I didn't understand the language in this comment and the one below that "this is the thing to match". Could you rephrase that, or consider dropping these inline comments?

The comments above the sections already pretty clearly describe the purpose.